assets/scss/_styles_project.scss

```diff
@@ -492,6 +497,19 @@ END MARKETING CSS
 
 /* END CSS FOR EXPANDER */
 
+/* START board carousel styling */
+
+.board-carousel {
+  display: block;
+  max-width: 80%;
```

Comment:
```suggestion
```

This is confusing but essentially the framework does some magic in changing how much width the content can take up. The easiest way to account for that is to put your content into a div like this:

```html
<div class="container td-max-width-on-larger-screens">
...
</div>
```